src/ontology/uberon-edit.obo

```diff
@@ -224854,6 +224854,23 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
+name: aorta-gonad-mesonephros region
+def: "Embryonic region that contains the dorsal aorta, genital ridges and mesonephros. It lies between the notochord and the somatic mesoderm, extending from the umbilicus to the anterior limb bud. This region is the first autonomous embryonic site for definitive haematopoiesis." [https://en.wikipedia.org/wiki/Aorta-gonad-mesonephros]
```

Comment:
I don't know if im comfortable wit the wording "contains the dorsal aorta, genital ridges and mesonephros" as I assume it contains structures that become those regions? correct me if I'm wrong - in which case the structure does not technically contain it

Reply:
"This region is the first autonomous embryonic site for definitive hematopoiesis" 
can this be interpreted as it having the axiom "'site_of' some hemopoiesis"? 
I ask as having that axiom will classify this as a hematopoietic tissue which I assume it is

Reply:
@shawntanzk I think that the region contains these embryonic structures: embryonic aorta, genital ridges and mesonephros (all embryonic structures)
@shawntanzk OK to add: axiom "'site_of' some hemopoiesis"